cache/store.go

```diff
@@ -53,45 +58,64 @@ func (a *kvItem) Less(b btree.Item) bool {
 	return a.key < b.(*kvItem).key
 }
 
-func (s *store) Get(startKey, endKey []byte) ([]*mvccpb.KeyValue, int64, error) {
+func (s *store) Get(startKey, endKey []byte, requestedRev int64) ([]*mvccpb.KeyValue, int64, error) {
```

Comment:
```suggestion
func (s *store) Get(startKey, endKey []byte, rev int64) ([]*mvccpb.KeyValue, int64, error) {
```

You don't call `startKey` as `requestedStartKey` so rev should not be prefixed either.